ui/app/mirrors/create/cdc/schemabox.tsx

```diff
@@ -245,6 +245,11 @@ export default function SchemaBox({
   const engineOptions = [
     { value: 'CH_ENGINE_REPLACING_MERGE_TREE', label: 'ReplacingMergeTree' },
     { value: 'CH_ENGINE_MERGE_TREE', label: 'MergeTree' },
+    {
+      value: 'CH_ENGINE_REPLICATED_REPLACING_MERGE_TREE',
+      label: 'ReplicatedReplacingMergeTree',
+    },
+    { value: 'CH_ENGINE_REPLICATED_MERGE_TREE', label: 'ReplicatedMergeTree' },
```

Comment:
I'm thinking whether we should add a tooltip for new users.
Maybe simply saying that works with ch cloud and what works with oss clusters

Reply:
mirror validation could check for host being CHC

Reply:
should be possible with `CheckIfClickHouseCloudHasSharedMergeTreeEnabled`

Reply:
That only runs when we're in clickpipes. Validation here is for OSS peerdb users with CHC destination

Reply:
yes I mean we can use that function outside of current codepath to get if a service is SMT which should mean we disable these engines

Reply:
I think for now we can omit this, people using CHC should know this engine doesn't apply to them